docs/Tutorials/CCE.md

```diff
@@ -90,8 +89,12 @@ following datasets:
 - `DrLapse.dat`
 - `DtLapse.dat`
 
-In this format, each row must start with the time stamp, and the remaining
-values are the complex modes in m-varies-fastest format. That is,
+#### Spherical harmonic modes {#cartesian_spherical_modes}
+
+In this format the spherical harmonic coefficients are stored. We use spherical
```

Comment:
The first sentence kind of trails off.

Reply:
I'm not sure what you are asking for here. I need an actionable change request in order to address anything. The sentence seems fine to me, maybe missing a comma after "format". Do you want me to say "coefficients of the quantities" or something?

Reply:
Oh I see what you were going for now. I read it expecting something else after "stored". What do you think about

```
In this format, the worldtube data are stored as spherical harmonic coefficients.
```